src/common/interfaces.ts

```diff
@@ -117,29 +117,25 @@ export interface IBBox {
 
 export type Footprint = Polygon | MultiPolygon | Feature<Polygon | MultiPolygon>;
 
-export interface PartSourceContext {
+export type PolygonFeature = Feature<Polygon, { maxZoom: number }>;
+
+export interface TilesSource {
```

Comment:
use this interface within the `UnifiedPart`

Reply:
changed to this type:
`export type UnifiedPart = {
  footprint: Feature<Polygon | MultiPolygon>;
  extent: BBox;
} & TilesSource;`